### DISCUSSION TITLE
How do you change the password?

### DISCUSSION THREAD

**Discussion by projx** [dec0dOS/zero-ui#16] [Q&A] [answered]
Hi

I fired up the docker-compose.yml without any issue, but I've since run into an issue, how do I change the password?  Doing it in the docker-compose.yml has no impact after its initial run.

Thanks

**Comment by dec0dOS** [comment, marked as answer]
Hello, @projx! Currently, the user management feature hasn't been committed to the main branch. Please clean the db.json file to reinitialize the password setup.

**Reply by projx** [reply]
Thanks, that appears to do the trick... Removed it, then restarted the container via docker-compose... Also another thing to note (for future users), don't put the username or password into "".

**Reply by zimage** [reply]
For clarification, I had to delete the entire "users" key from the db.json file before restarting in order for the new credentials to work.

**Comment by pingscloud** [comment]
That didn't worked for me. i have cleared the file `/var/lib/docker/volumes/root_zero-ui_data/_data/db.json` and did `docker restart zu-main` but the default user returns back

**Comment by squromiv** [comment]
Any news?

**Comment by KravenDarklore** [comment]
Hey i'm sorry. I edited the username and password in the yml file, and deleted the entire user section out of db.json.

Could you please explain what exactly I need to delete, and how to get the new password to work because it keeps reverting back.

**Reply by dec0dOS** [reply]
You should edit the files when the services are down (`docker-compose down`) it is going to revert back.

**Comment by darksidelemm** [comment]
Is there a way of changing the password on an 'established' instance of zero-UI? 
Can a hash be generated somehow and updated in the db.json file?

**Reply by dec0dOS** [reply]
Just set the new password in the docker-compose.yml and remove the hash from db.json file

User management is in progress

**Reply by darksidelemm** [reply]
OK, and this will not overwrite any of the other settings? (e.g. network configurations)

**Reply by ghost** [reply]
How are you getting on with the user management control? @dec0dOS 
Trying to control the function that stops brute force attacks. By only allowing 3 password attempts for a period of time, but it just won’t accept the edit, it just simply does nothing?
